test/validation/api/ipsec/ipsec_test_out.c

```diff
@@ -1138,6 +1138,127 @@ static void test_out_ipv4_udp_esp_null_sha256(void)
 	ipsec_sa_destroy(sa);
 }
 
+static void test_sa_info(void)
+{
+	uint32_t src = IPV4ADDR(10, 0, 111, 2);
+	uint32_t dst = IPV4ADDR(10, 0, 222, 2);
+	odp_ipsec_tunnel_param_t tunnel_out;
+	odp_ipsec_tunnel_param_t tunnel_in;
+	odp_ipsec_sa_param_t param_out;
+	odp_ipsec_sa_param_t param_in;
+	odp_ipsec_sa_info_t info_out;
+	odp_ipsec_sa_info_t info_in;
+	odp_ipsec_capability_t capa;
+	odp_ipsec_sa_t sa_out;
+	odp_ipsec_sa_t sa_in;
+
+	CU_ASSERT_EQUAL(0, odp_ipsec_capability(&capa));
+
+	memset(&tunnel_out, 0, sizeof(tunnel_out));
+	memset(&tunnel_in, 0, sizeof(tunnel_in));
+
+	tunnel_out.type = ODP_IPSEC_TUNNEL_IPV4;
+	tunnel_out.ipv4.src_addr = &src;
+	tunnel_out.ipv4.dst_addr = &dst;
+
+	ipsec_sa_param_fill(&param_out,
+			    false, false, 0x4a2cbfe3, &tunnel_out,
+			    ODP_CIPHER_ALG_AES_GCM, &key_mcgrew_gcm_3,
+			    ODP_AUTH_ALG_AES_GCM, NULL,
```

Comment:
The algorithms could be AES-CBS-128 and HMAC-SHA1-96 as those are mandatory for every IPsec implementation (same change was proposed in PR 1127).

Reply:
Ack. I had to add the same check function introduced in PR 1127 - ipsec_check_esp_aes_cbc_128_sha1 here as well for this. Once the PR 1127(IPsec Print) is merged, I can drop the definition of ipsec_check_esp_aes_cbc_128_sha1() function from this PR.